usaspending_api/conftest.py

```diff
@@ -123,6 +123,15 @@ def django_db_setup(
             ensure_business_categories_functions_exist()
             call_command("load_broker_static_data")
 
+            # temporarily updating the environment so commands in tests will point to the test database
+            old_environ = os.environ.copy()
+            os.environ["DATABASE_URL"] = generate_test_db_connection_string()
```

Comment:
I think maybe we should be more precise here, and only store the single values we changed, and then only restore those single values. While the env _probably shouldn't_ be changing permanently during the course of a test run ... who knows. 

So could this be a dict that captures any environ value we change, and then in the finalizer, the dict's values are reset into the os.environ?